src/components/application_manager/src/commands/mobile/on_system_request_notification.cc

```diff
@@ -77,25 +77,22 @@ void OnSystemRequestNotification::Run() {
     return;
   }
 
-  if (RequestType::PROPRIETARY == request_type) {
-    /* According to requirements:
-       "If the requestType = PROPRIETARY, add to mobile API fileType = JSON
-        If the requestType = HTTP, add to mobile API fileType = BINARY"
-       Also in Genivi SDL we don't save the PT to file - we put it directly in
-       binary_data */
-
-    const std::string filename =
-        (*message_)[strings::msg_params][strings::file_name].asString();
-    BinaryMessage binary_data;
-    file_system::ReadBinaryFile(filename, binary_data);
+  const std::string& filename =
+      (*message_)[strings::msg_params][strings::file_name].asString();
+  BinaryMessage binary_data;
+  file_system::ReadBinaryFile(filename, binary_data);
 #if defined(PROPRIETARY_MODE)
-    AddHeader(binary_data);
+  AddHeader(binary_data);
```

Comment:
@LitvinenkoIra 
Its applicable only for PROPRIETARY request type, but not for every type

Reply:
@dev-gh Fixed 0362e4a3e11bb4267143a1978d8b33687fdfb136